.travis.yml

```diff
@@ -1,11 +1,15 @@
-sudo: false
+sudo: required
```

Comment:
this was some problem with using trusty for tests. Just pushed a new commit reverting this to see what happens